src/pages/iou/request/IOURequestStartPage.tsx

```diff
@@ -161,7 +149,6 @@ function IOURequestStartPage({
                             {iouType !== CONST.IOU.TYPE.SEND ? (
                                 <OnyxTabNavigator
                                     id={CONST.TAB.IOU_REQUEST_TYPE}
-                                    selectedTab={selectedTab || CONST.IOU.REQUEST_TYPE.SCAN}
```

Comment:
why this is removed ?

Reply:
The `OnyxTabNavigator` component doesn't accept this prop. It is loaded using `withOnyx` inside the component.